packages/xdl/src/UserSettings.ts

```diff
@@ -107,11 +107,16 @@ async function anonymousIdentifier(): Promise<string> {
   return id;
 }
 
+function userToken(): string | null {
```

Comment:
Maybe `accessToken` would be a more accurate name for this field? And perhaps then use the same name everywhere (i.e. `authorizationToken` -> `accessToken`.

Reply:
Done!